nebula-components/tools/_triangle.scss

```diff
@@ -1,32 +1,27 @@
-@mixin nb-triangle($direction, $size: $nb-flyout-triangle-size, $color: $nb-flyout-triangle-color) {
+@mixin nb-triangle(
+  $direction,
+  $color: $nb-color-white,
+  $transform: rotate(-45deg),
+  $size: ($nb-spacing-unit / 2),
+  $box-shadow-color: $nb-color-grey-light) {
   content: '';
   display: block;
   position: absolute;
   height: 0;
   width: 0;
+  transform-origin: 0 0;
+  transform: #{$transform};
+  border-width: $size;
+  border-style: solid;
   @if ($direction == up) {
-    bottom: 100%;
-    border-bottom: $size solid $color;
-    border-left: $size solid transparent;
-    border-right: $size solid transparent;
+    bottom: 96.75%;
```

Comment:
Think we need to take a look at these magic numbers, they're _very_ brittle as they are.